web/dev.ts

```diff
@@ -334,8 +325,6 @@ export async function setupDevDB() {
     s6.isInstructor = false;
     s6.isProjectCoach = true;
```

Comment:
don't we have to remove this too?

Reply:
Yeah, we could also remove them. Will do in some followup PR, this is already large enough. We also need to discuss whether we drop the data from the database.